src/Pyz/Zed/Console/ConsoleDependencyProvider.php

```diff
@@ -179,9 +179,6 @@
 use Spryker\Zed\Uuid\Communication\Console\UuidGeneratorConsole;
 use Spryker\Zed\ZedNavigation\Communication\Console\BuildNavigationConsole;
 use Spryker\Zed\ZedNavigation\Communication\Console\RemoveNavigationCacheConsole;
-use SprykerSdk\Spryk\Console\SprykBuildConsole;
-use SprykerSdk\Spryk\Console\SprykDumpConsole;
-use SprykerSdk\Spryk\Console\SprykRunConsole;
```

Comment:
@stereomon Those are not used right?
So removing them here was just never done?
I wonder why CS didnt fail due to not needed use statements then.

Reply:
CI should fail on those true.

Reply:
This is suite and I double-checked now. I guess those were wrongly removed same as for the commands. Suite uses `spryker-sdk/spryk:0.3.0` which is fine and public we should not remove.